projects/angular/src/data/stack-view/_stack-view.clarity.scss

```diff
@@ -199,154 +70,77 @@
       // This way we keep a border-bottom on the last element if the stack view is higher
       // than the total height of the  blocks it contains, but we do not have a
       // weird 1px scroll when both heights match exactly.
-      @include lastElementBottomBorder(variables.$clr-use-custom-properties);
+      border-bottom: none;
     }
 
-    .stack-block-changed {
-      > .stack-block-label {
-        margin-left: -1 * variables.$clr_baselineRem_0_375;
```

Comment:
Can't find what happened with this one?

Reply:
New visual test is created for it [stack-view_clrSbNotifyChange](https://1071--storybook-clarity-design.netlify.app/?path=/story/stack-view-stack-block--default&args=clrSbNotifyChange:true). 

Originally the stack view row label was pushed right by the triangle in the picture, but now the triangle have `position: absolute;` and the negative margin is not needed any more. 

![image](https://github.com/vmware-clarity/ng-clarity/assets/6692908/5adc3f9c-60b2-4153-85ec-94269c2bb5cf)